Tracking/TrackLength/src/TrackLengthProcessor.cc

```diff
@@ -66,19 +68,14 @@ void TrackLengthProcessor::processEvent(EVENT::LCEvent * evt){
         streamlog_out(DEBUG9)<<std::endl<<"Starting to analyze "<<i+1<<" PFO"<<std::endl;
         ReconstructedParticle* pfo = static_cast <ReconstructedParticle*> ( pfos->getElementAt(i) );
 
-        int nClusters = pfo->getClusters().size();
-        int nTracks = pfo->getTracks().size();
-
-        if( nClusters != 1 || nTracks != 1){
-            // Analyze only simple pfos. Otherwise write dummy zeros
+        if( pfo->getTracks().empty() ){
+            // Calculate track length only for particles with at least one track.
+            // Note: If the pfo has more than one track attached (e.g. kink, v0), only the first track will be used. The results may be not accurate in these cases. 
```

Comment:
```suggestion
            // Note: If the pfo has more than one track attached (e.g. kink, v0), only the first track will be used. The results may be not accurate in these cases. 
            streamlog_out(DEBUG9) << "PFO has no tracks. Writing zeros." << std::endl;
```

Might be a useful debug message to have.